ibc-testkit/src/relayer/context.rs

```diff
@@ -140,12 +143,13 @@ mod tests {
             .host_type(HostType::SyntheticTendermint)
             .latest_height(chain_b_start_height)
             .build()
-            .with_client_parametrized_with_chain_id(
-                chain_id_a,
-                &client_on_b_for_a,
-                client_on_b_for_a_height,
-                Some(mock_client_type()), // The target host chain is mock.
-                Some(client_on_b_for_a_height),
+            .with_client_config(
+                MockClientConfig::builder()
+                    .client_chain_id(chain_id_a)
+                    .client_id(client_on_b_for_a.clone())
+                    .client_state_height(client_on_b_for_a_height)
```

Comment:
A side suggestion: Let's rename the `client_state_height` to the `latest_height`

Reply:
[b02520e](https://github.com/cosmos/ibc-rs/pull/1051/commits/b02520ee3dc8b81668bd1cdd28cda2876248f3e3) & [e4e0730](https://github.com/cosmos/ibc-rs/pull/1051/commits/e4e073059ffedb853938116f9433302263eff305)